benchmarks/clhstackbench.c

```diff
@@ -46,6 +54,9 @@ inline static Object pop(int pid) {
         n = (Node *)Top;
         Top = Top->next;
     }
+#ifdef DEBUG
+    stack_state += 1;
```

Comment:
`stack_state` is increased both during push and pop, so this counter shows the number of total operations, right?

Reply:
Exactely